src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_IVariableDeclaration.cs

```diff
@@ -610,9 +610,9 @@ void M1()
             var compilation = CreateEmptyCompilation(source);
             (var operation, _) = GetOperationAndSyntaxForTest<VariableDeclaratorSyntax>(compilation);
             var declarator = (IVariableDeclaratorOperation)operation;
-            Assert.Equal(2, declarator.Children.Count());
-            Assert.Equal(OperationKind.Literal, declarator.Children.First().Kind);
-            Assert.Equal(OperationKind.VariableInitializer, declarator.Children.ElementAt(1).Kind);
```

Comment:
❔ Fairly obvious question, but we do still have some tests for `Children` right? 😄

Reply:
I'm not particularly worried about having dedicated tests for this API. It's deprecated and forwards directly to a well-tested version.